include/votca/xtp/gw.h

```diff
@@ -125,8 +125,7 @@ class GW {
     }
     double deriv(double frequency) const {
       return _sigma_c_func.CalcCorrelationDiagElementDerivative(_gw_level,
-                                                                frequency) +
-             _offset - frequency;
+                                                                frequency);
```

Comment:
```suggestion
                                                                frequency)-1.0; // -1.0 comes from derivative of the frequency itself.
```

Reply:
How? This is just the freq derivative of Sigma?

Reply:
ok, this is used in the grid solver for the root finding problem, when `f(w)=Sigma_c(w) + w0 + Sigma_x - Vxc - w = 0`

But then, the `qpweight` is calculated from this as as `1/(1 - f'(w)) = 1/(1-Sigma_c(w)' +1)`, no? That should be correcte to `1/(-f'(w))`, I guess.

Reply:
I do not remember what the quasiparticle weight definition is.

Reply:
`1/(1-Sigma')`

Reply:
true.

Reply:
then you mention is correct.